content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,125 @@
+OpenTelemetry.registerMeterProvider(meterProvider: MeterProviderBuilder()
+            .with(resource: DefaultResource.get())
+            .build())
+```
+
+### Details
+
+`SDKResourceExtension` provides attributes in a resource object with details
+about the iOS device, OS details, and application details. It applies these
+values to the appropriate
+[semantic attributes](/docs/reference/specification/resource/semantic_conventions/#semantic-attributes-with-sdk-provided-default-value).
+
+#### Application Info
+
+| Attribute           | Value example                 | Description                                                                                              |
+| ------------------- | ----------------------------- | -------------------------------------------------------------------------------------------------------- |
+| `service.name`      | `MyApplication`               | `CFBundleName`; The application name defined in the App's info.plist.                                    |
+| `service.version`   | `1.0 (1234)`                  | `CFBundleShortVersion` & (`CFBundleVersion`); The application version as defined in the App's info.plist |
+| `service.namespace` | `com.myCompany.myApplicaiton` | `CFBundleIdentifier`                                                                                     |
+
+#### Device Info
+
+| Attribute                 | Value example           | Description                                    |
+| ------------------------- | ----------------------- | ---------------------------------------------- |
+| `device.model.identifier` | `iphone13,3`            | fetched from `sysctl` depending on device type |
+| `device.id`               | `00000000-0000-0000000` | `identifierForVendor` uuid string              |
+
+#### Operating System Info
+
+| Attributes       | Value example                     | Description                                            |
+| ---------------- | --------------------------------- | ------------------------------------------------------ |
+| `os.type`        | `darwin`                          | predefined in `ResourceAttributes`                     |
+| `os.name`        | `iOS`, `watchOS`, `macOS`         | `UIDevice.current.systemName` or dependent on platform |
+| `os.version`     | `15.4.0`                          | `ProcessInfo.processInfo.operatingSystemVersion`       |
+| `os.description` | `iOS Version 15.4 (Build 19E240)` | A combination of os name, version and build.           |
+
+## `NSURLSession` Instrumentation
+
+This instrumentation creates spans for all network requests made with
+NSURLSessions. It also injects distributed tracing headers in instrumented
+network requests. `NetworkStatus` is a dependency of this package, which
+provides network status attributes on network spans.
+
+### Usage
+
+Initialize the class with
+`URLSessionInstrumentation(configuration: URLSessionInstrumentationConfiguration())`
+to automatically capture all network calls.
```

Comment:
Could this be followed be a copy-pasteable code sample? Nothing wild or serving a specific example, just something someone can copy-paste and tweak, or at least see an example of how that initialization is best done.